_posts/2019-05-10-why-projects-need-codenames.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Why Projects Need Code Names
+date: 2019-05-10
+author: Joey Aghion
+categories: [engineering, git, github]
+---
+
+Before I joined Artsy, I worked at companies where projects tended to have meaningful, predictable names. If we were building a system for flagging media uploads, it might be called `media-review`. In many cases, our code repositories' names matched the main product's branding or even the company's name. Life was simple and there was no risk of ambiguity.
```

Comment:
From reading on down below, one thing that might be good to clarify: is there an implicit size/scope assumption here? A system (even if it is small) is different than a library, and a library can also be a project. Might be useful to describe the boundary a bit (or lack thereof) as lead-in to the post.

Reply:
Good point. I think I'm talking mostly about code repositories and systems that will be operated as opposed to libraries or non-code projects. Will modify...

Reply:
I added "software project" more explicitly in a few spots and called out libraries for public release.